hc/accounts/tests/test_login.py

```diff
@@ -18,14 +17,14 @@ def test_it_sends_link(self):
 
         r = self.client.post("/accounts/login/", form)
         assert r.status_code == 302
-
-        ### Assert that a user was created
-
+        ### **Assert that a user was created
+        final_list = len(User.objects.all())
+        self.assertEqual(1, final_list)
         # And email sent
         self.assertEqual(len(mail.outbox), 1)
         self.assertEqual(mail.outbox[0].subject, 'Log in to healthchecks.io')
         ### Assert contents of the email body
-
+        self.assertIn("please open the link below:", mail.outbox[0].body)
```

Comment:
@collinmutembei 
This assert checks for the content inside the page HTML, meaning if in future or for some reason the content of the page is changed e.g the string changes to "Find a link to log in below", this test fails not because the functionality is broken but the page changed. Would it be the case to assert that the content was returned by checking the status code?